### ISSUE TITLE
rultor merge failed: "no space left on device" 

### ISSUE THREAD

**Issue by dmzaytsev** [yegor256/rultor#906]
rultor merge failed  with message 

>    at com.github.klieber.phantomjs.install.WebInstaller.install(WebInstaller.jtime="2015-07-20T15:09:58Z" level="fatal" msg="write /dev/stdout: no space left on device" 

see details here http://www.rultor.com/t/4569-122912557

**Comment by AbdealiJK** [comment]
Im getting the same.

```
 fatal: Unable to create '/tmp/rultor-vHeJ/repo/.git/refs/remotes/fork/sils/ccpp.lock': No space left on device error: Could not fetch fork
```
Log : http://www.rultor.com/t/4572-122920575
PR : https://github.com/coala-analyzer/coala/pull/836

**Comment by dmzaytsev** [comment]
@AbdealiJK try again, should work

**Comment by AbdealiJK** [comment]
Yep, works for me now

**Comment by dmzaytsev** [comment]
@yegor256 I'm closing the ticket

**Comment by sils1297** [comment]
On long term you might want to think about how to prevent something like
this in the future? (Or did you do this already?)

2015-07-21 10:23 GMT+02:00 Dmitry Zaytsev <notifications@github.com>:

**Comment by dmzaytsev** [comment]
@yegor256 it's not a reproducible bug that's why I closed it.
but I agree with @sils1297, please take it into account

**Comment by sils1297** [comment]
Well you could probably do something like send out urgent emails to the
maintainers *before* rultor runs out of space or do cleanups periodically.

2015-07-21 10:39 GMT+02:00 Dmitry Zaytsev <notifications@github.com>:

**Comment by yegor256** [comment]
@dmzaytsev yes, definitely, @sils1297 is right. Would be great to prevent this from happening and notify me immediately. Please, reopen the ticket and we'll dispatch it

**Comment by dmzaytsev** [comment]
@yegor256 I reopen the ticket

**Comment by alex-palevsky** [comment]
@dmzaytsev I added milestone `2.0` to this issue, let me know if there has to be something else

**Comment by alex-palevsky** [comment]
@yegor256 it is your task now

**Comment by alex-palevsky** [comment]
@dmzaytsev thank you for reporting this, I added 15 mins to your acc, transaction `61663663`

**Comment by yegor256** [comment]
@alex-palevsky assign someone else pls

**Comment by alex-palevsky** [comment]
> @alex-palevsky assign someone else pls

@yegor256 I deducted 30 points from your rating

**Comment by alex-palevsky** [comment]
> @alex-palevsky assign someone else pls

@yegor256 someone else will help in this task, no problem at all

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

, as his depends on #942

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.
> 
> , as this depends on #942 

@original-brownbear yep, got it, let's wait for #942 being solved

**Comment by alex-palevsky** [comment]
@dmzaytsev impediment in issue #942 is closed, we can probably try to continue here

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear got it, "postponed" label here

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear all right, we'll find someone else for this task

**Comment by original-brownbear** [comment]
@alex-palevsky this depends on #1055

**Comment by alex-palevsky** [comment]
> @alex-palevsky this depends on #1055

@original-brownbear OK, let's wait for #1055

**Comment by alex-palevsky** [comment]
@dmzaytsev #1055 is closed, we were waiting for it.. maybe we can continue?

**Comment by 0crat** [comment]
I can't do it for technical reasons, I'm very sorry. If you don't know what to do, submit this error as a ticket [here](https://github.com/zerocracy/datum):

```
1.0-SNAPSHOT 0d912b4 2017-12-27T21:47:39
IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/finish_order.groovy
java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/finish_order.groovy
	at com.zerocracy.farm.reactive.StkGroovy.process(StkGroovy.java:125)
	at com.zerocracy.farm.StkSafe.process(StkSafe.java:87)
	at com.zerocracy.farm.reactive.StkSmart.process(StkSmart.java:66)
	at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:82)
	at com.zerocracy.farm.reactive.Brigade.lambda$apply$0(Brigade.java:64)
	at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:58)
	at org.cactoos.func.IoCheckedBiFunc.apply(IoCheckedBiFunc.java:66)
	at org.cactoos.func.UncheckedBiFunc.apply(UncheckedBiFunc.java:60)
	at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:59)
	at org.cactoos.iterator.Filtered.hasNext(Filtered.java:90)
	at org.cactoos.iterator.LengthOf.value(LengthOf.java:56)
	at org.cactoos.iterable.LengthOf.value(LengthOf.java:56)
	at com.zerocracy.farm.reactive.Brigade.apply(Brigade.java:67)
	at com.zerocracy.farm.r...
```

Cc @yegor256

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/rultor#906` was not in scope
